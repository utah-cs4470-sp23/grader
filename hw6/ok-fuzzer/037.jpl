show (- [765][(- (if false then 218 else (- 857)))])

