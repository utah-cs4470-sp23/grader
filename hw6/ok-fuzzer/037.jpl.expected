(ShowCmd
 (UnopExpr
  (IntType)
  -
  (ArrayIndexExpr
   (IntType)
   (ArrayLiteralExpr (ArrayType (IntType) 1) (IntExpr (IntType) 765))
   (UnopExpr
    (IntType)
    -
    (IfExpr
     (IntType)
     (FalseExpr)
     (IntExpr (IntType) 218)
     (UnopExpr (IntType) - (IntExpr (IntType) 857)))))))
